profile/README.md

```diff
@@ -57,7 +57,12 @@ repos for [Atsign's technology](https://docs.atsign.com/).
 
 ## We are proud to sponsor
 
-<!-- pyml disable-num-lines 3 md013,md033-->
-<img height=120px src="https://atsign.com/wp-content/uploads/2023/03/eff-2023-member-org.png.webp" alt="EFF Org Member">
-<img height=120px src="https://atsign.com/wp-content/uploads/2021/10/GEC-graphics-01-1.png.webp" alt="Global Encryption Coalition">
-<img height=120px src="https://atsign.com/wp-content/uploads/2023/08/LoRa-Alliance-horizontal600x300.png" alt="Lora Alliance">
+<!-- pyml disable-num-lines 8 md012,md013,md033-->
```

Comment:
md012 is "no inline html" for the `<div>`

The div makes the images spread horizontally instead of vertically